### PR TITLE
Bump docker to v27.2.0

### PR DESCRIPTION
**Pull request by aabian**
Bump docker dependency to `27.2.0`

### CODE REVIEW

**Comment by aabian** [comment]
In case you are reviewing with an older version of docker engine installed locally, you could try:

```
DOCKER_API_VERSION=1.46 make acceptance
```

File: acceptance/go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module gitlab.otters.xyz/product/journey/library/aceptadora.git/acceptance
 go 1.22
 
 require (
-	github.com/cabify/aceptadora v0.0.0-00010101000000-000000000000
```
Comment (yaroslavcev) [inline acceptance/go.mod]: I think this pseudo version was intentionally set to highlight that we use current version with replace statement later.
```
replace github.com/cabify/aceptadora => ../
```

Having particular tag might confuse reader, making him thinking that we run acceptance on an older version

**Comment by yaroslavcev** [comment]
> In case you are reviewing with an older version of docker engine installed locally, you could try:
> 
> ```
> DOCKER_API_VERSION=1.46 make acceptance
> ```

Acceptance [failed](https://github.com/cabify/aceptadora/actions/runs/10669631627/job/29595540534) with
```
Error response from daemon: client version 1.47 is too new. Maximum supported API version is 1.45
```
Could you try adding `DOCKER_API_VERSION=1.45` to https://github.com/cabify/aceptadora/blob/003770ba73ab5c4513bf24fe0808717a7661d901/acceptance/config/default.env

File: acceptance/go.mod (hunk 1 above)
Comment (aabian) [reply acceptance/go.mod]: Thanks for the comment, I've added your suggestion

**Review by yaroslavcev — APPROVED** [review]

**Pull request merged**